config/jobs/etcd/etcd-bbolt-presubmits.yaml

```diff
@@ -6,6 +6,7 @@ presubmits:
       always_run: true
       branches:
         - main
+        - release-1.4
```

Comment:
We also have bbolt release-1.3

Reply:
Yes, I'm aware. We still have some legwork to finalize migrating the rest of the projects. With our limited resources, I've been focusing on the `etcd` repository first.